microsoft-edge/webview2/samples/wv2deploymentwixcustomactionsample.md

```diff
@@ -138,7 +138,7 @@ If you want the app to download the Evergreen WebView2 Runtime Bootstrapper (`Mi
 
```

Comment:
in edge-developer\microsoft-edge\webview2\concepts\frames.md
reword 
if `ExecuteScript` is called before the `ContentLoading` event is raised, the script will not be **executed** and the string `null` will be returned.

Reply:
fixed locally